src/git.ts

```diff
@@ -0,0 +1,122 @@
+import * as github from "@actions/github";
+import { Octokit } from "@octokit/rest";
+import * as core from "@actions/core";
+import parseDiff, { File as DiffFile } from "parse-diff";
+import { ReviewComment } from "./types";
+
+const GITHUB_TOKEN = core.getInput("GITHUB_TOKEN");
```

Comment:
Consider using `process.env.GITHUB_TOKEN` instead of `core.getInput('GITHUB_TOKEN')` for better security and to avoid potential issues with input handling.